processor/src/main/java/org/mapstruct/ap/internal/model/source/MappingOptions.java

```diff
@@ -37,7 +37,7 @@
  */
 public class MappingOptions extends DelegatingOptions {
 
-    private static final Pattern JAVA_EXPRESSION = Pattern.compile( "^java\\((.*)\\)$" );
+    private static final Pattern JAVA_EXPRESSION = Pattern.compile( "^\\s*java\\((.*)\\)\\s*$", Pattern.DOTALL );
```

Comment:
I guess that @Zegveld made a remark here during our meeting..

Reply:
Yea he did, he had a better idea for how to do this. Without using a `Pattern`, but I'll leave that improvement to him 😄